ad-insertion-button/input.py

```diff
@@ -89,70 +91,76 @@ def setupScte(device_ip, duration):
 	if "usb" in str(device.phys):
 
 		if "input0" in str(device.phys):
-			print(device.path, device.name, device.phys)
+			#print(device.path, device.name, device.phys)
 			deviceCap = device.capabilities()
-			print(deviceCap)
+			#print(deviceCap)
 			keyEventPath = device.path
 
 dev = InputDevice(keyEventPath)
 
 print(dev)
 
+# Splice Insert with a preroll of 0 seconds and duration of 30 seconds (This is changed depending on button press)
+splice = {"splice_command":{"value":"splice_insert","splice_insert":{"preroll_time_msec":0,"duration_msec":30000}}}
+# Time Signal
+time_signal_start = {"splice_command":{"value":"time_signal","time_signal":{"preroll_time_msec": 200,"segmentation_descriptor":"021C435545494800008E7FCF0000A4CB8008080000000000000000340200"}}}
```

Comment:
For clarity of this code sample, it would be good to add a comment and give details about the `segmentation_descriptor` values.